draft-irtf-cfrg-vdaf.md

```diff
@@ -1219,7 +1219,7 @@ discrete Fourier transform, as this allows for fast polynomial interpolation.
 (One example is Prio3 ({{prio3}}) when instantiated with the generic FLP of
 {{flp-generic-construction}}.) Specifically, a field is said to be
 "FFT-friendly" if, in addition to satisfying the interface described in
-{{field}}, it implements the following method:
+{{fields}}, it implements the following method:
```

Comment:
I think we still want to point to the section above, not the field parameters table.